### DISCUSSION TITLE
Error message when loading images from API

### DISCUSSION THREAD

**Discussion by Robert-Schurig** [neuefische/questions#356] [Web] [answered]
Hello everyone,

I am trying to display images that I fetched from an API on my page. I checked on the console that the connection works. Showing the title of an image works, displaying images does not work.
I committed and pushed the latest version. [You can find it here](https://github.com/Robert-Schurig/Capstone-App/pull/6) The branch is called apiconnection.

The error appears on opening the page on the localhost. I expect to see quite a few images on this page.

 I tried different ways to format the src property and I followed the instructions from this discussion: https://github.com/orgs/neuefische/discussions/239 I changed the next.config.js but if I refresh the page I still get this error message:
![Bildschirm­foto 2022-12-01 um 11 56 51](https://user-images.githubusercontent.com/114500583/205047291-ba4a96f5-426d-4ab9-a2f1-ad10e19e55ed.png)

What might be the solution to get the images displayed? Thanks a lot for your help.

**Comment by pixelass** [comment, marked as answer]
Please take a look at the configuration options: https://nextjs.org/docs/api-reference/next/image#configuration-options

You need to add the domain, so that nextjs knows that you consider this source as secure.
